libvcx/src/aries/handlers/connection/connection.rs

```diff
@@ -359,12 +485,18 @@ impl Connection {
      */
     pub fn connect(&mut self) -> VcxResult<()> {
         trace!("Connection::connect >>> source_id: {}", self.source_id());
+        let pairwise_info = self.pairwise_info();
+        let cloud_agent = CloudAgentInfo::create(&pairwise_info)?;
+        let routing_keys = cloud_agent.routing_keys()?;
+        let agency_endpoint = cloud_agent.service_endpoint()?;
+        self.cloud_agent_info = cloud_agent;
```

Comment:
Did we agree to deprecate this? (We can do it separately of course...)

Reply:
yes I agree, I'll do that separately

Reply:
OK